web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/ProductV2Controller.java

```diff
@@ -0,0 +1,52 @@
+package it.pagopa.selfcare.dashboard.web.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import it.pagopa.selfcare.dashboard.web.model.ExchangedToken;
+import it.pagopa.selfcare.dashboard.web.security.ExchangeTokenServiceV2;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/v2/products", produces = MediaType.APPLICATION_JSON_VALUE)
+@Api(tags = "products")
+public class ProductV2Controller {
+
+    private final ExchangeTokenServiceV2 exchangeTokenService;
+
+    @Autowired
+    public ProductV2Controller(ExchangeTokenServiceV2 exchangeTokenServiceV2) {
+        this.exchangeTokenService = exchangeTokenServiceV2;
+    }
+
+    @GetMapping(value = "/{productId}/back-office")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.dashboard.product.api.retrieveProductBackoffice}")
```

Comment:
you could add nickname to set operationId, for ex: "v2RetrieveProductBackofficeUsingGET"